src/EB1170.c

```diff
@@ -36,15 +129,49 @@ ApiStatus func_802406E0_EB1540(Evt* script, s32 isInitialCall) {
     return ApiStatus_DONE2;
 }
 
-INCLUDE_ASM(s32, "EB1170", func_8024077C_EB15DC);
+ApiStatus func_8024077C_EB15DC(Evt *event) {
+    s32 *readPos;
+
+    readPos = event->ptrReadPos;
+    event->varTable[2] = evt_get_variable(event, *readPos++);
+    event->varTable[3] = evt_get_variable(event, *readPos++);
+    event->varTable[4] = evt_get_variable(event, *readPos++);
+    event->varTable[5] = evt_get_variable(event, *readPos++);
+    event->varTable[6] = evt_get_variable(event, *readPos++);
+    event->varTable[7] = evt_get_variable(event, *readPos++);
+    event->varTable[8] = evt_get_variable(event, *readPos++);
+    event->varTable[9] = evt_get_variable(event, *readPos++);
+    event->varTable[10] = evt_get_variable(event, *readPos++);
+    event->varTable[11] = evt_get_variable(event, *readPos++);
+    return ApiStatus_DONE2;
+}
+
+
 
 //INCLUDE_ASM(s32, "EB1170", func_8024086C_EB16CC);
```

Comment:
I know you didn't do this in your PR, but that include_asm comment can be removed. I'll let brotenko review the rest of this

Reply:
Removed locally - I'll include it with the commit for the rest of brotenko's feedback.